internal/services/network/web_application_firewall_policy_resource.go

```diff
@@ -272,11 +275,41 @@ func resourceWebApplicationFirewallPolicy() *pluginsdk.Resource {
 													Required:     true,
 													ValidateFunc: validate.ValidateWebApplicationFirewallPolicyRuleGroupName,
 												},
-												"disabled_rules": {
+												"rule": {
```

Comment:
this is a breaking change?

Reply:
No, `disabled_rules` is still kept in current version and will be replaced by `rule` in 4.0.  The `disabled_rules` is dynamically added at the end of this function controlled by `if !features.FourPointOhBeta()`.

Reply:
I see that now! thanks